sdv/constraints/tabular.py

```diff
@@ -379,3 +381,73 @@ def reverse_transform(self, table_data):
         table_data[self._column] = self._formula(table_data)
 
         return table_data
+
+
+class Rounding(Constraint):
+    """Round a column based on the specified number of digits.
+
+    Args:
+        column (list[str]):
+            Name of the column(s) to round.
+        digits (int or dict[str->int]):
```

Comment:
What happens if the dict and the `column` list differ? I would validate that and raise a `ValueError` if they do